content/html/concepts/semantic-html/terms/figcaption/figcaption.md

```diff
@@ -30,7 +30,7 @@ For example:
 ```html
 <figure>
   <img src="overwatch.jpg">
-  <figcaption>This picture shows characters from Overwatch.</figcaption>
+  <figcaption>This picture shows characters from overwatch.</figcaption>
```

Comment:
Hi @TemitopeAgbaje, Overwatch is a game. So let's keep this capitalized.

Reply:
Feel free to edit this to say:
 
`<figcaption>This picture shows characters from the video game Overwatch.</figcaption>`

Reply:
Okat @sonnynomnom  would correct that back